xcode/Subconscious/Shared/Components/Detail/MemoEditorDetail.swift

```diff
@@ -1690,6 +1690,15 @@ struct MemoEditorDetailModel: ModelProtocol {
         return Update(state: model, fx: fx)
     }
     
+    static func tryWikilink(link: EntryLink) -> String {
```

Comment:
Does this function get used anywhere?

Reply:
Oops, no, dead code from a refactor. Will drop it.